docs/content/how-to/kubevirt/create-kubevirt-cluster.md

```diff
@@ -47,16 +78,21 @@ operator is installed, it is now possible to create a guest cluster.
 Below is an example of how to create a guest cluster using environment
 variables and the `hypershift` cli tool.
 
+!!! note
+
+    The --release-image flag could be used to provision the HostedCluster with a specific OpenShift Release (the hypershift operator has a support matrix of releases supported by a given version of the operator)
+
+
 ```shell linenums="1"
-export CLUSTER_NAME=example
+export HOSTEDCLUSTER_NAME=example
```

Comment:
The reason we use CLUSTER_NAME instead of HOSTEDCLUSTER_NAME is that `CLUSTER_NAME` is consistent with documentation for all the other hypershift platforms. I'd prefer to remain consistent. So if we're going to change this env var, i'd want to change it across the board.

Reply:
Changed back to CLUSTER_NAME.